docs/sql-reference/sql-functions/string-functions/group_concat.md

```diff
@@ -2,70 +2,92 @@
 
 ## Description
 
-This is an aggregate function similar to sum(). group_concat concatenates non-null values into one string, with the second argument `sep` being the separator. The second argument can also be omitted. This function usually needs to be used along with `group by`.
-
-> Please note that strings may not be concatenated in sequence because it uses distributed computing.
+group_concat concatenates non-null values into one string from a group, with a separator argument, which is ',' by default if not specified.
 
 ## Syntax
 
-```Haskell
-VARCHAR group_concat(VARCHAR str[, VARCHAR sep])
+```SQL
+VARCHAR GROUP_CONCAT([DISTINCT] expr [,expr ...]
+             [ORDER BY {unsigned_integer | col_name | expr}
+                 [ASC | DESC] [,col_name ...]]
+             [SEPARATOR str_val])
 ```
 
 ## Parameters
 
-- `str`: the values to concatenate. It must evaluate to VARCHAR.
-- `sep`: the separator, optional. If it is not specified, `, ` (a comma and a space) is used by default.
+- `expr`: the values to concatenate, ignoring null. It should be cast to VARCHAR. They can be optionally specified `DISTINCT` to eliminate duplicate values. More `expr` are concatenated directly, use `concat()` or `concat_ws` to specify formats.
+- order-by items can be unsigned integers (identify `expr`, starting from 1), column names or normal expressions. To sort in reverse order, add the DESC (descending) keyword to the name of the column you are sorting by in the ORDER BY clause. The default is ascending order; this may be specified explicitly using the ASC keyword
+- `str_val`: the optional separator is used to concat non-null values from different rows. If it is not specified, `,` (a comma) is used by default.
 
 ## Return value
 
-Returns a VARCHAR value.
+Returns a string value for each group, but returns NULL if there are no non-NULL values.
+
+set `group_concat_max_len` to limit the length of output string from a group, its default value is 1024, minimal value is 4.
```

Comment:
Give an example to explain how to use this?

Reply:
added